src/leetCodeSolutionProvider.ts

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) jdneo. All rights reserved.
+// Licensed under the MIT license.
+
+import * as hljs from "highlight.js";
+import * as MarkdownIt from "markdown-it";
+import * as path from "path";
+import * as vscode from "vscode";
+import { Disposable, ExtensionContext, ViewColumn, WebviewPanel, window } from "vscode";
+import { IProblem } from "./shared";
+
+class LeetCodeSolutionProvider implements Disposable {
+
+    private context: ExtensionContext;
+    private panel: WebviewPanel | undefined;
+    private markdown: MarkdownIt;
+    private markdownPath: string; // path of vscode built-in markdown extension
+    private solution: Solution;
+
+    public initialize(context: ExtensionContext): void {
+        this.context = context;
+        this.markdown = new MarkdownIt({
+            linkify: true,
+            typographer: true,
+            highlight: this.codeHighlighter.bind(this),
+        });
+        this.markdownPath = path.join(process.env.VSCODE_CWD as string, "resources", "app", "extensions", "markdown-language-features");
```

Comment:
Use `vscode.env.appRoot` instead of the environment variable since it will not always be available

Reply:
Yes, you are right. The VSCODE_CMD does not exist when I debug on my Linux machine.